src/6.7/6.7.2.0.md

```diff
@@ -0,0 +1,412 @@
+
+### App user change
+
+API requests can now be executed in the context of a specific app user so that Change history & audit logs show the correct user. Additionally, permissions and ownerships can be respected properly.
+
+#### How to use it
+
+You can now specify the user context for API calls when your App authenticates, for example via manifest.xml. Typically, this is done by sending the `sw-context-token` or an OAuth token linked to the desired user. Add the header `sw-app-user: <user-id>` or use the API token tied to that user, so that any API call like `POST /api/product` runs under that user's name. 
+
+In **Administration → Settings → System → Users & Permissions**, each user has a UUID. That UUID is used in the API request header to switch context.
+
+### Storefront OpenAPI schema
+
+Extended with additional GET methods.
+
+1. Store API Product Listing (/store-api/product-listing/{categoryId})
+
+* What it does: This is the core endpoint for fetching a paginated list of products for a given category. It handles all the complex filtering, sorting, and association loading needed for a category page. Before this, the OpenAPI schema likely lacked a complete definition for this crucial endpoint. Documenting it properly allows frontend developers and third-party integrators to understand the exact request and response structure for category navigation.
+
+2. Store API Product Detail (/store-api/product/{productId})
+
+What it does: This endpoint retrieves all the detailed information for a specific product, including its variants, media, prices, configurator settings, and more. Similar to the listing endpoint, formally adding this to the schema provides a official contract for the data returned on a product detail page, which is essential for headless commerce implementations.
+
+### Symfony profiler
+
+Complete cart data like line items, price calculations, transactions etc. can now be observed in Symfony profiler toolbar. Now, you can get a complete, clean, and organized view of the cart's state on any page load without interrupting the execution flow, dramatically speeding up development and troubleshooting.
+
+### Cart rule loader extension event
+
+The `RuleLoaderEvents::LOADER_EXTENSION_EVENT` is a powerful new hook that allows developers to bypass the limitations of the visual rule builder. It injects custom, PHP-based business logic directly into the process that determines which promotions are applied to the cart, greatly expanding the possibilities for creating sophisticated and dynamic promotions.
+
+This event is an "extension point." It doesn't let you change the rules themselves, but it allows you to add entirely new conditions or logic on top of the existing rule system.
+
+The classic problem it solves, for example, could be 
+
+* Discount applies if it's the user's birthday
+
+### Custom route names
```

Comment:
this change is rather an edge case and totally not what is documented here

Before the was a hardcoded list of prefixes your routes need to have for storefront routes e.g. `frontend` or `widget`, now you can add your own prefixes via config, but this change is mostly aethetically